src/types.ts

```diff
@@ -9,6 +9,10 @@ export type WeekDayFormat = 'narrow' | 'short' | 'long'
 
 export type PickerOptions = Partial<ResolvedLocaleOptions>
 
+export type PickerOptionsWithResolvedCalendar = Omit<PickerOptions, 'calendar'> & {
```

Comment:
I don't understand why did you have to create a new type here? calendar can still be undefined. We can change the `ResolvedLocaleOptions` `calendar` to be Temporal.CalendarProtocol if that's more appropriate

Reply:
This would create a lot of ts errors and it is true only for nepali. Other calendars have ResolvedLocaleOptions[calendar] as `string`.